### PR TITLE
kvrpc: add raw scan.

### PR DESCRIPTION
**Pull request by disksing**
cc @siddontang @AndreMouche

### CODE REVIEW

File: proto/kvrpcpb.proto [hunk 1]
```diff
@@ -226,3 +226,14 @@ message RawDeleteResponse {
...
+
+message RawScanRequest {
+    Context context = 1;
+    bytes start_key = 2;
+    uint32 limit = 3;
+}
+
+message RawScanResponse {
+    errorpb.Error region_error = 1;
+    repeated KvPair kvs = 2;
```
Comment (siddontang) [inline proto/kvrpcpb.proto]: How user uses a start key to start another scan request?

Comment (AndreMouche) [inline proto/kvrpcpb.proto]: May be we could add the following two element in response:

* `IsTruncated` Specifies whether (true) or not (false) all of the results were returned
* `NextStartKey` indicate next start_key when `IsTruncated` is true.

Comment (disksing) [inline proto/kvrpcpb.proto]: NextStartKey = `append(lastKey, \0)`

Comment (disksing) [inline proto/kvrpcpb.proto]: And I don't think we need _IsTruncated_, client can decide whether to start another scan by checking if `len(kvs) == limit`.

Comment (siddontang) [inline proto/kvrpcpb.proto]: I think NextStartKey is enough if the user finds NextStartKey is none, he/she can think the scan is over.

**Comment by disksing** [comment]
PTAL @AndreMouche @siddontang

**Comment by siddontang** [comment]
LGTM

**Pull request merged**